def a <- f(10) handle
    err => print "Something went wrong"
